internal/dashboard/api.go

```diff
@@ -89,21 +88,17 @@ func (d *DashboardAPI) GetDashboard(c *gin.Context) {
 
 	i := 0
 	for _, c := range clusters {
-		status, err := c.GetStatus()
 		if err == nil {
-			if strings.ToUpper(status.Status) == "RUNNING" {
-				go func() {
-					logger := d.logger.WithField("clusterId", c.GetID())
-					cluster, partial := d.getClusterDashboardInfo(logger, c, organizationID)
-					if partial {
-						partialResponse = true
-					}
-					clusterResponseChan <- cluster
-				}()
-				i++
-			}
+			go func(commonCluster cluster.CommonCluster) {
```

Comment:
Shouldn't we allow filtering by status as well? (e.g. the status to filter by could be passed in as query parameter)

Reply:
for now we said we can show clusters in all statuses, no need to filter